<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsBasicWidget</class>
 <widget class="QWidget" name="SettingsBasicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basic Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Language:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="languageComboBox">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Appereance:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="styleComboBox">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>System Settings</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Light</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dark</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Startup:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Download Splits:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="threadCountSpinBox">
       <property name="maximum">
        <number>512</number>
       </property>
       <property name="singleStep">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Automatic upgrade:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="autoUpgradeCheckBox">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Default Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="savePathLineEdit">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectPathButton">
       <property name="text">
        <string>Select(&amp;S)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QCheckBox" name="notifyOnFinishedCheckBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Notify after download completion</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="autoContinueCheckBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Automatically resume all unfinished tasks</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Task Management:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="autoStartupCheckBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Open at login</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
